storage/books.json

```diff
@@ -0,0 +1,94 @@
+[
```

Comment:
- [ ] Please make sure that the app still works properly when the JSON files don't exist. If the storage directory doesn't exist, the following error appears. Kindly fix the error and make sure that the data is stored in a new JSON file if there isn't one.

![image](https://github.com/iankiema/Ruby-capstone-project/assets/62290545/4ae15667-df37-44f6-84bf-1ee493e4a0a0)